opentelemetry-sdk/src/logs/log_emitter.rs

```diff
@@ -249,14 +249,17 @@ impl opentelemetry::logs::Logger for Logger {
     fn emit(&self, mut record: Self::LogRecord) {
         let provider = self.provider();
         let processors = provider.log_processors();
-        let trace_context = Context::map_current(|cx| {
-            cx.has_active_span()
-                .then(|| TraceContext::from(cx.span().span_context()))
-        });
 
         //let mut log_record = record;
-        if let Some(ref trace_context) = trace_context {
-            record.trace_context = Some(trace_context.clone());
+        if record.trace_context.is_none() {
```

Comment:
This emitter behavior could benefit from a test, but wasn't sure where to add it.